include/ametsuchi/pager/page.h

```diff
@@ -34,21 +38,49 @@ enum class SpanningItem : uint8_t {
   SINGLE_INFIX
 };
 
-
 struct PageDirectory {
   uint16_t offset;
   uint16_t length;
 };
 
+struct FileHeader {
```

Comment:
it's at the first page, probably default implementation should be out of this file